integration-libs/epd-visualization/root/epd-visualization-root.module.ts

```diff
@@ -0,0 +1,42 @@
+import { APP_INITIALIZER, NgModule } from '@angular/core';
+import {
+  CmsConfig,
+  ConfigInitializerService,
+  provideConfigValidator,
+  provideDefaultConfigFactory,
+} from '@spartacus/core';
+import { epdVisualizationConfigValidator } from './config';
+import { EPD_VISUALIZATION_FEATURE } from './feature-name';
+
+export function epdVisualizationJsFactory(
```

Comment:
This APP_INITIALIZER seems doing nothing. Do we really need it?

Reply:
I will check if it can be removed.